### PR TITLE
Inputs validation

### LINKED ISSUE #1141: Inputs should have 2 properties (valid, invalid) instead of one (valid)
Properites should be true/false only

### PR DESCRIPTION
**Pull request by anna-wro**
* true/false → valid, invalid
Closes #1141

### CODE REVIEW

File: src/components/form-elements/Select.jsx [hunk 1]
```diff
@@ -2,25 +2,19 @@ import React from 'react';
...
     value,
     options = [],
     ...additionalProps
   } = props;
 
   const selectClass = classnames('sg-select', {
-    'sg-select--valid': valid === VALIDATION.VALID,
-    'sg-select--invalid': valid === VALIDATION.INVALID,
+    'sg-select--valid': valid,
+    'sg-select--invalid': invalid,
```
Comment (aju) [inline src/components/form-elements/Select.jsx]: check if both are true throw error
@kdzwinel @matikrk @vergilius

Comment (matikrk) [inline src/components/form-elements/Select.jsx]: 👍 
console.error()

Comment (kdzwinel) [inline src/components/form-elements/Select.jsx]: rather `throw new Error("bla")`?

Comment (aju) [inline src/components/form-elements/Select.jsx]: @kdzwinel it can crash application
maybe console.error is better option?

Comment (kdzwinel) [inline src/components/form-elements/Select.jsx]: if both will be true then it's a bug and we should crash IMO

Comment (aju) [inline src/components/form-elements/Select.jsx]: yeah, lets go with the throw!

Comment (anna-wro) [inline src/components/form-elements/Select.jsx]: Done!

Comment (matikrk) [inline src/components/form-elements/Select.jsx]: imho  better is console error instead of throw whole application.

**Pull request merged**